dnf/module/exceptions.py

```diff
@@ -0,0 +1,92 @@
+# supplies the 'module' command.
```

Comment:
Is it wise to store some of exceptions on different place than in dnf/exceptions.py?

Reply:
Yes.